### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by dengemann**
hope it fixes the dinosauric inria build bot.

cc @agramfort @Eric89GXL

### CODE REVIEW

File: mne/tests/test_cov.py [hunk 1]
```diff
@@ -343,7 +356,7 @@ def test_cov_scaling():
     assert_true(cov.max() < 1)
 
 
-@requires_sklearn
+@requires_recent_sklearn
```
Comment (agramfort) [inline mne/tests/test_cov.py]: wouldn't be nicer to pass the version as param to the decorator?

Comment (dengemann) [inline mne/tests/test_cov.py]: indeed. but this is a fix not a fearture  PR

File: mne/viz/evoked.py [hunk 2]
```diff
@@ -511,9 +511,10 @@ def whitened_gfp(x, rank=None):
 
     ax = ax_gfp[0]
     if n_columns == 1:
-        ax.legend(loc='upper right', bbox_to_anchor=(0.98, 0.9), fontsize=12)
+        ax.legend(  # mpl < 1.2.1 compatibility: use prop
+            loc='upper right', bbox_to_anchor=(0.98, 0.9), prop=dict(size=12))
     else:
-        ax.legend(loc='upper right', fontsize=10)
+        ax.legend(loc='upper right', prop=dict(size=10))
```
Comment (agramfort) [inline mne/viz/evoked.py]: explain why this syntax with a comment

Comment (dengemann) [inline mne/viz/evoked.py]: I added a comment above

**Comment by agramfort** [comment]
merged by rebase. Thanks @dengemann !

let's see if buildbot is happy.

**Pull request merged**